pkg/runtime/wfengine/backends/actors/backend.go

```diff
@@ -412,7 +412,7 @@ func (abe *ActorBackend) String() string {
 
 func (abe *ActorBackend) validateConfiguration() error {
 	if abe.actorRuntime == nil {
-		return errors.New("actor runtime has not been configured")
+		return errors.New("actor runtime has not been configured. Ensure Placement is running")
```

Comment:
added this for for clarity, see discord user facing this and it [not being clear placement was not running](https://discord.com/channels/778680217417809931/1075836407156850698/1283744676884512871)

Reply:
nitpick: "... Placement service ..."
but is this clarified in terms of the scheduler backend?

Reply:
fair point, I've changed it to this: `actor runtime has not been configured. Ensure actor backend is running, either Placement service or Scheduler service`. I dont want to bubble up which is being used bc then that changes the code to know which system is being used, but its clear now for users to check them. Rather then having users spend time to figure out they need to check placement or scheduler.